tinyagentos/routes/system_logs.py

```
@@ -0,0 +1,501 @@
+) -> dict:
+    # journalctl returns oldest-first within a "-n <lines>" window; there is
+    # no direct "entries before this cursor" flag, so this v1 pager treats
+    # `before` as a resume token via --after-cursor (a simple, if not fully
+    # bidirectional, paging scheme) and reverses the page so the response is
+    # newest-first, as the design calls for.
+    args = ["-u", unit, "--no-pager", "-o", "json", "-n", str(lines)]
+    if before:
+        args += ["--after-cursor", before]
+    try:
+        rc, stdout = await _run_journalctl(args, timeout=10.0)
+    except Exception as exc:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)